types/requests.ts

```diff
@@ -245,3 +245,81 @@ export interface CodeExecutionTool {
    */
   codeExecution: {};
 }
+
+/**
+ * Request message for generating image.
+ * @public
+ */
+export interface ImageGenerationRequest {
```

Comment:
We do not want an option to choose the model? 

There is Imagen 3 and Imagen 3 Fast. how one can choose between them? 

btw to which one imagen-3.0-generate-001 refers to? as they have different pricing.